### ISSUE TITLE
Reference Paper

### ISSUE THREAD

**Issue by idevede** [ippqw5/PINNLearning#1]
Hi @ippqw5,

Thanks for the well-organized repo! I would like to ask about the paper you used for the Jupyter " 7_11_Parabolic耦合pde模型.ipynb". You can directly email me with the address: cdf@pku.edu.cn. 

I am looking forward to your response!

Best,
Ide

**Comment by ippqw5** [comment]
“The Numerical solution of the coupled viscous Burgers’ equation”.
It is placed in the folder "04.论文资料"

**Comment by ippqw5** [comment]
PARTITIONED TIMESTEPPING FOR A PARABOLIC TWO DOMA.pdf
It is placed in the folder “04.论文资料”.

**Comment by idevede** [comment]
Hi @ippqw5,

Thanks for the response!

I am still not sure how PINN can satisfy the schema of solving the coupled PDE. It would be helpful if you can provide more references or intuitions. :)

Best,
Ide

**Comment by ippqw5** [comment]
Hi @idevede ，
I'm sorry for my misunderstanding!
I just used the model from the paper "PARTITIONED ..." to implement PINNs.

My strategy is to create two separate neural networks, each representing the respective solution of the two regions.
In each epoch, the same training points on Interface are imported into the two networks respectively, so we can obtain the loss_Interface by automatic differentiation. 

In addition, the above two neural networks also need to transmit different training points of  corresponding regions. So, we can get the  loss through pde equations in single region. (In this problem, we have two regions)

Finally, these losses are added to obtain the total LOSS, which means that we can train the above two networks simultaneously by LOSS。

I'm sorry for my poor English,and I hope you can understand my explanation.

The structure of the network is shown by following picture.
![image](https://user-images.githubusercontent.com/72686792/192080837-9659c7a9-1c6a-4560-9634-e0c62f5f07df.png)

**Comment by idevede** [comment]
Hi @ippqw5,

Thanks for the clear response! If it is convenient for you, we can add Wechat friend, and feel free to email me with your Wechat ID. :)

Best,
Ide
